### PR TITLE
New Article: Cloud Backup - Basic troubleshooting on Linux Servers

### PR DESCRIPTION
**Pull request by BadCrumpany**
New article created to assist less technical customers with initial troubleshooting of Cloud Backups on their Cloud Server.

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c37d530c20/how-to/).

**Comment by cyrichardson** [comment]
@BadCrumpany Just checking in to see if you are waiting for a review on this article or if it is on hold.

**Comment by BadCrumpany** [comment]
Hello!  This was a new article so just waiting on someone to review it.  Thanks!

**Comment by cyrichardson** [comment]
@BadCrumpany Reviewing now!

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b4bac4fc9d/how-to/)

**Comment by cyrichardson** [comment]
@BadCrumpany I've made some editorial changes. Please review the file and give any feedback that you have.

**Comment by cyrichardson** [comment]
@BadCrumpany Here's the staging link to your new article: https://staging.developer.rackspace.com/support.rackspace.com/build-bee79111ca/how-to/cloud-backup-basic-troubleshooting-on-linux-servers/

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-48ff28800b/how-to/).

**Comment by BadCrumpany** [comment]
Hi @cyrichardson thanks for the work on this.  I just updated two characters and everything looks great.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d3ac3010d8/how-to/)

**Comment by cyrichardson** [comment]
OK, I'll ask for the final review/merge in #docs Slack channel now.

File: content/cloud-servers/cloud-backup-basic-troubleshooting-on-linux-servers.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+are caused by the backup agent being disconnected on the Linux&reg; instance, including
+the following problems:
+
+* Backups have not been running.
+* Backups are taking too long.
+* Cloud Backup is taking up all of the server’s resources.
+
+As a Managed Infrastructure customer who administers your own servers, this article helps
+you diagnose if the server is experiencing an issue that you can remedy, or if you should
+escalate the issue to a member of Rackspace Support. This article provide instructions with
```
Comment (catlook) [inline content/cloud-servers/cloud-backup-basic-troubleshooting-on-linux-servers.md]: ... article provides instructions ...

File: content/cloud-servers/cloud-backup-basic-troubleshooting-on-linux-servers.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+helpful information that you can use to troubleshoot the issue or include in a ticket when
+you create onet.
```
Comment (catlook) [inline content/cloud-servers/cloud-backup-basic-troubleshooting-on-linux-servers.md]: ... create one.

File: content/cloud-servers/cloud-backup-basic-troubleshooting-on-linux-servers.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+## Check for open issues
+
+Before you restart the DriveClient service, check [status.rackspace.com](https://status.rackspace.com)
+to ensure that there are no open issues for Cloud Backup that could be impacting your server instance.
+In addition, check the current support tickets to ensure that there has not been an incident
+that is causing the lack of accessibility of the server.
```
Comment (catlook) [inline content/cloud-servers/cloud-backup-basic-troubleshooting-on-linux-servers.md]: ... lack of server accessibility.

**Pull request merged**